frappe/config/integrations.py

```diff
@@ -77,6 +77,11 @@ def get_data():
 					"name": "OAuth Provider Settings",
 					"description": _("Settings for OAuth Provider"),
 				},
+				{
+					"type": "doctype",
+					"name": "Connected App",
+					"description": _("Connected App"),
+				},
```

Comment:
I don't know if this is needed. If it is, please ensure description is same as that of the Desk Page.

Reply:
Not sure as well. If it's not, the whole file could be removed, right?